### PR TITLE
M0090IDE-3910 JRE must not be used to install and launch the SDK

### PR DESCRIPTION
**Pull request by denisbedin**

### CODE REVIEW

File: SDKUserGuide/systemRequirements.rst [hunk 1]
```diff
@@ -12,21 +12,21 @@ System Requirements
...
-
-      macOS with Apple M1 chip is not officially supported. Please contact :ref:`our support team <get_support>` for more details. 
+   - macOS with Apple M1 chip (tested on version 12.0.1 Monterey)
 
 - **Java :**
-   - JRE or JDK 8 (Oracle JDK or other OpenJDK build: tested on AdoptOpenJDK/Eclipse Adoptium)
+   - JDK 11 (Oracle JDK or other OpenJDK build: tested on AdoptOpenJDK/Eclipse Adoptium)
+
+.. warning::
+   A full JDK is required to run Eclipse, so please ensure that you do not use a JRE to install and launch the SDK.
```
Comment (tdelhomenie-microej) [inline SDKUserGuide/systemRequirements.rst:21]:
```suggestion
   Starting from the SDK Distribution 22.06, a JDK is required to install and launch it, so please ensure that you do not use a JRE.
```

**Review by tdelhomenie-microej — CHANGES_REQUESTED** [review]

**Comment by denisbedin** [comment]
@tdelhomenie-microej  the suggestion is applied

**Review by tdelhomenie-microej — APPROVED** [review]

**Comment by tdelhomenie-microej** [comment]
@denisbedin LGTM (to be merged after the SDK release)

File: SDKUserGuide/systemRequirements.rst [hunk 2]
```diff
@@ -3,22 +3,54 @@
...
+   :widths: auto
+   :header-rows: 1
+
+   * - Distribution
+     - SDK
+     - Architecture
+     - JRE/JDK
+   * - 22.06
+     - 5.6.0
+     - > 7.17
```
Comment (denisbedin) [inline SDKUserGuide/systemRequirements.rst:32]: Shouldn't the architecture' version be minimum 7.17 instead of >7.17 ?

Comment (tdelhomenie-microej) [reply SDKUserGuide/systemRequirements.rst]: Right, good catch. Done.

**Pull request merged**